lib/thumbs/pull_request_worker.rb

```diff
@@ -130,9 +130,27 @@ def run_command_in_docker(command)
       [output, exit_code]
     end
 
+    def otp_installations
+      output, exit_code = `kerl list installations`
+      installations={}
+      output.lines.each do |line|
+        build_name, path = line.split(/\s+/)
+        installations[build_name] = path
+      end
+      installations
+    end
+
     def run_command_in_shell(command)
       shell=thumb_config['shell']||'/bin/bash'
-
+      if thumb_config.key?('otp')
+        build_name=thumb_config['otp']
+        unless otp_installations.key?(build_name)
+          output =  "OTP #{thumb_config.key?('otp')} not present."
```

Comment:
If we consider the question, "What do I do now that _otp_version_ is not present?", we can do better, either:

1. provide a means to install other otp versions, list installed otp versions, ..., basically putting a front-end on kerl.
2. provide the list of installed otp versions in the error message and a prompt to contact DevOps to install more, if needed.
3. document the list of installed otp versions in the README or wiki and respond w/ read the manual.